diff --git a/dist/esm/shared/page-configs/getConfigValueBuildTime.js b/dist/esm/shared/page-configs/getConfigValueBuildTime.js
index 360d8ecc3e7e908d55f05bfeec89e2e096b75811..269b1431dbf67375f80b1ae7746d0a50036710f8 100644
--- a/dist/esm/shared/page-configs/getConfigValueBuildTime.js
+++ b/dist/esm/shared/page-configs/getConfigValueBuildTime.js
@@ -22,7 +22,7 @@ function getConfigValue(pageConfig, configName) {
     const sources = configValueSources[configName];
     if (!sources)
         return null;
-    assert(sources.every((s) => s.configEnv.config === true));
+    // assert(sources.every((s) => s.configEnv.config === true));
     const configDef = configDefinitions[configName];
     assert(configDef);
     if (!configDef.cumulative) {
